File: fizzbuzz/FizzBuzz.java
public class FizzBuzz {
    private static void fizzBuzz(int n) {
        for (int i = 1; i < n + 1; ++i) {
            System.out.print(i + " ");
            if (i % 3 == 0)
                System.out.print("fizz");
            if (i % 5 == 0)
                System.out.print("buzz");
            System.out.println("");
        }
    }
    public static void main(String[] args) {
        fizzBuzz(100);
    }
}

File: fizzbuzz/fizzbuzz.cpp
#include <iostream>

void fizzBuzz(int n);

int main() {
    fizzBuzz(100);
}

void fizzBuzz(int n) {
    for (int i = 1; i < n + 1; ++i) {
        printf("%d ", i);
        if (i % 3 == 0)
            printf("fizz");
        if (i % 5 == 0)
            printf("buzz");
        printf("\n");
    }
}